# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libv4l/hotfix.patch.musl
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- v4l-utils-1.26.0/utils/meson.build.vanilla	2023-11-30 21:11:55.918495626 +0100
+++ v4l-utils-1.26.0/utils/meson.build	2023-11-30 21:12:12.763496712 +0100
@@ -36,7 +36,7 @@
 subdir('v4l2-ctl')
 subdir('v4l2-dbg')
 subdir('v4l2-sysfs-path')
-subdir('v4l2-tracer')
+#subdir('v4l2-tracer')
 
 # Compliance tools
 subdir('cec-compliance')
--- v4l-utils-1.26.0/utils/v4l2-tracer/v4l2-tracer-common.h.vanilla	2023-11-30 21:04:37.445467338 +0100
+++ v4l-utils-1.26.0/utils/v4l2-tracer/v4l2-tracer-common.h	2023-11-30 21:11:19.292493263 +0100
@@ -6,6 +6,7 @@
 #ifndef V4L2_TRACER_COMMON_H
 #define V4L2_TRACER_COMMON_H
 
+#include <sys/types.h>
 #include "v4l2-info.h"
 #include "codec-fwht.h"
 #include "media-info.h"
@@ -25,7 +26,6 @@
 #include <string.h>
 #include <sys/ioctl.h>
 #include <sys/mman.h>
-#include <sys/types.h>
 #include <unistd.h>
 #include <unordered_map>
 #include <vector>
